app/assets/javascripts/helpers.js.coffee

```diff
@@ -20,7 +20,14 @@ _.extend PopHealth.Helpers,
     else
       return value
 
-
+  formatNumber: (value) ->
```

Comment:
I'm not sure we want this function. popHealth already includes Numeral.js and uses it in the handlebars templates. Numeral.js can already do what this function is intended to do.